packages/jss/src/renderers/DomRenderer.js

```diff
@@ -255,10 +256,10 @@ const getNonce = memoize(
 )
 
 const insertRule = (
-  container: CSSStyleSheet | CSSKeyframesRule | CSSMediaRule,
+  container: CSSStyleSheet | CSSMediaRule | CSSKeyframesRule,
   rule: string,
   index?: number = container.cssRules.length
-): false | Object => {
+): false | any => {
```

Comment:
Is there no way around `any` here?

Reply:
Ah, no this was temp change, but it doesn't matter when we update to the latest flow version, Object will equal any

Reply:
I think we can actually do much better types now, many anys are just there because of initial lack of experience with flow